docs/tutorials/aqt/getting_started.ipynb

```diff
@@ -0,0 +1,268 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "0341a89e3dc0"
+   },
+   "source": [
+    "##### Copyright 2020 Alpine Quantum Technologies GmbH"
```

Comment:
This needs to be:

```suggestion
    "##### Copyright 2020 The Cirq Developers"
```

AQT is part of The Cirq Developers automatically being a contributor to the repo.